mpf/platforms/fast/fast_serial_communicator.py

```diff
@@ -121,7 +122,7 @@ async def _identify_connection(self):
             self.remote_processor, self.remote_model, self.remote_firmware = (
                 msg[3:].split())
         except ValueError:
```

Comment:
Can we have an unit test for this part? I would like a test where we use the fallback path.

Reply:
I would even more like a firmware update so the segment display returns the same formatted response as the other platforms :) 

But yes, I can add one

Reply:
The segment controller should be temporarily returning "ID:SEG FP-DSP-2050-1 00.50\r" which I believe matches the existing FAST stuff, but if not I can certainly modify it.

Reply:
Yeah, looks like that fallback is not needed. I've removed it :)